server/mentionscrawler/authentication/authenticate.py

```diff
@@ -0,0 +1,14 @@
+from flask import request, current_app
+from .token import decode_token
+from jwt.exceptions import InvalidTokenError
+
+
+def authenticate(request_body: object):
+    if "token" in request_body:
+        try:
+            decode_token(current_app.secret_key, request_body["token"])
+            return True
+        except InvalidTokenError:
+            return False
+    else:
+        return False
```

Comment:
I'll have to look more about how function decorators work then because I basically had this same thing, but it had one less function in it (I didn't have def wrap(fn))